tests/std/tests/P0896R4_ranges_range_machinery/test.cpp

```diff
@@ -968,12 +968,8 @@ STATIC_ASSERT(!ranges::view<std::string_view const&>);
 
 STATIC_ASSERT(test_begin<std::span<int>, std::span<int>::iterator>());
 STATIC_ASSERT(test_end<std::span<int>, std::span<int>::iterator>());
-STATIC_ASSERT(test_cbegin<std::span<int>, std::span<int>::iterator>());

```

Comment:
There should be no changes necessary in this file; these asserts should all continue to pass. (The range access CPOs are smart enough to ignore member `cr?(begin|end)` in the first place.)

Reply:
@CaseyCarter  again you are too fast. My daughter was not yet asleep...

I wanted to actually ask you whether it is indeed intended that this compiles

```cpp
std::span<int> foo{someRange};
auto it = std::ranges::rbegin(foo);
```
but this does not
```cpp
std::span<int> foo{someRange};
auto it = std::rbegin(foo);
```

This feels like an icky design

Reply:
> but this does not

Sure it does; `std::cbegin` also ignores member `cbegin`. And `std::cbegin` no longer does something different than member `cbegin` for `span`.